go/Makefile

```diff
@@ -0,0 +1,49 @@
+solidity_dir=$(realpath ${SOLIDITY_DIR})
+solidity_files := $(foreach file,${SOLIDITY_FILES},$(file))
+
+# Solidity filenames without .sol with subdirectories prefixes.
+contract_stems := $(basename $(solidity_files))
+
+abi_files := $(addprefix abi/,$(addsuffix .abi,$(contract_stems)))
```

Comment:
Thoughts on adding the same comment as in `keep-ecdsa` and in `keep-core`? It helps to understand what's happening here:
```
# Go abigen bindings in abi/ subdirectory with .go suffix, alongside solc ABI
# files with .abi suffix.
```